src/search/screens/search.screen.js

```diff
@@ -155,6 +154,19 @@ class Search extends Component {
     this.renderItem = this.renderItem.bind(this);
   }
 
+  componentDidMount() {
+    this.setQueryAsLocation();
+  }
+
+  setQueryAsLocation = () => {
+    const { params } = this.props.navigation.state;
+
+    if (params) {
+      this.switchQueryType(1);
+      this.search(`location:${params.location}`, 1);
```

Comment:
Please use `SearchTypes.USERS`, instead of `1`. And you'd better check `params && params.location`.